core/vtk/ttkPersistenceDiagramClustering/ttkPersistenceDiagramClustering.h

```diff
@@ -0,0 +1,1021 @@
+#ifndef _TTK_PERSISTENCEDIAGRAMSCLUSTERING_H
```

Comment:
Thanks for providing doxygen entries for authorship and references to the related publication. See core/vtk/ttkTopologicalSimplification/ttkTopologicalSimplification.h for an example.
Thanks for reporting this also to the headers of the base layer.